test/cpu_load_balancing.bats

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bats
+# vim: set syntax=sh:
+
+load helpers
+
+function setup() {
+	export activation="cpu-load-balancing.crio.io"
+	setup_test
+	sboxconfig="$TESTDIR/sbox.json"
+	ctrconfig="$TESTDIR/ctr.json"
+}
+
+function teardown() {
+	cleanup_test
+}
+
+function check_sched_load_balance() {
+	local is_enabled="$1"
+
+	if is_cgroup_v2; then
+		return
+	else
+		loadbalance_path="/sys/fs/cgroup/cpuset"
+		loadbalance_filename="cpuset.sched_load_balance"
+	fi
+
+	loadbalance=$(cat "$loadbalance_path/pod_123.slice/$loadbalance_filename")
+	[[ "$is_enabled" == *"$loadbalance"* ]]
+}
+
+@test "test cpu load balancing" {
+	cpuset="0-1"
+
+	start_crio
```

Comment:
a workload needs to be created that enabled the allowed_annotation, otherwise it will be denied. see `test/workloads.bats`